contracts/contracts/token/OUSD.sol

```diff
@@ -234,32 +252,90 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         address _to,
         uint256 _value
     ) internal {
-        bool isNonRebasingTo = _isNonRebasingAccount(_to);
-        bool isNonRebasingFrom = _isNonRebasingAccount(_from);
+        if (_from == _to) {
+            return;
+        }
```

Comment:
nit: I feel like this is probably unnecessary. I don't see a case where someone would expect transfer to self to work or not fail. Could just revert here.

Reply:
cc @DanielVF curious if there is any special reason behind why to return rather to revert